var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -60,6 +64,8 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
 
     conflicts('+cuda', when='+rocm', msg="CUDA and ROCm are mutually exclusive")
     conflicts('+rocm', when='@:2.0', msg="ROCm support is available in DBCSR v2.1 and later")
+    conflicts('+cuda', when='+opencl')
+    conflicts('+rocm', when='+opencl')
```

Comment:
conflict needed for version

Reply:
forgot that we don't have versions yet in the DBCSR Spack package, sorry!
We can therefore also drop the `when='@:2.0'` from above I made you put in there in the last PR.